### ISSUE TITLE
some error with bokeh_events_test_4.py 

### ISSUE THREAD

**Issue by cx994** [jdzakary/PDB_Energy_Analysis#13]
Hi @jdzakary . When running the example bokeh_events_test_4.py , I found the scatter plot does not appear. Did you meet the same problem?

**Comment by jdzakary** [comment]
Hi @cx994. I should clarify that the testing folder is not code tests like you would use in Django, etc. It's really more of a collection of experiment files that I was using to learn how streamlit's bokeh interface worked. None of the files are necessary for the project to run. I will either rename the folder or remove it to dispel confusion.

I did encounter the same issue in bokeh_events_test_4.py, and that is ultimately why I did not use the "streamlit_bokeh_events" package (notice that it is not in project requirements.txt). It seems that the package is broken, or at least cannot properly display an object created by "bokeh.plotting.figure". Notice that it works well in "bokeh_events_test" and "bokeh_events_test_3" where objects from the bokeh.models module are used.

It may be possible to find a way to create the scatterplot without using bokeh.plotting.figure, which then would hopefully work with the "streamlit_bokeh_events" package.
